web/components/input/buttons/RadioGroup/RadioButton.sass

```diff
@@ -0,0 +1,54 @@
+.radio-button
+  margin-left: 20px;
+  font-family: Apercu Pro;
+  font-size: 18px;
+  font-weight: bold;
+  line-height: 23px;
+  color: $primary;
+  position: relative;
+  margin-bottom: 8px;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+
+  &:hover input:enabled ~ .radio-img
+    background-image: url('../assets/radio/hover.svg');
+
+  input[type="radio"]
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+
+    &:checked ~ .radio-img
+      background-image: url('../assets/radio/selected.svg');
+      &:after
+        display: block;
+
+    &:disabled
+      cursor: not-allowed;
```

Comment:
Not a big fan of manipulating cursors - is the disabled style and unclickableness of an element not enough feedback?

Maybe @sherryyx could weigh in? (this is what causes the big `x` on the mouse that the demo gif shows)

Reply:
It was just something I added to make it extra clear that the input is disabled, @sherryyx it can be removed if you don't want it from a UX perspective.